metrics/bleu/README.md

```diff
@@ -2,35 +2,37 @@
+Scores are calculated for individual translated segments—generally sentences—by comparing them with a set of good quality reference translations. Those scores are then averaged over the whole corpus to reach an estimate of the translation's overall quality. Neither intelligibility nor grammatical correctness are not taken into account.
 
 ## Intended Uses
 BLEU and BLEU-derived metrics are most often used for machine translation.
 
 ## How to Use
 
-This metric takes as input lists of predicted sentences and reference sentences:
+This metric takes as input a list of predicted sentences and a list of lists of reference sentences (since each predicted sentence can have multiple references):
 
 ```python
->>> predictions = [
-...     ["hello", "there", "general", "kenobi",
-...     ["foo", "bar" "foobar"]
-... ]
+>>> predictions = ["hello there general kenobi", "foo bar foobar"]
 >>> references = [
-...     [["hello", "there", "general", "kenobi"]],
-...     [["foo", "bar", "foobar"]]
+...     ["hello there general kenobi", "hello there !"],
+...     ["foo bar foobar"]
 ... ]
 >>> bleu = evaluate.load_metric("bleu")
 >>> results = bleu.compute(predictions=predictions, references=references)
 >>> print(results)
-{'bleu': 0.6370964381207871, 'precisions': [0.8333333333333334, 0.75, 1.0, 1.0], 'brevity_penalty': 0.7165313105737893, 'length_ratio': 0.75, 'translation_length': 6, 'reference_length': 8}
+{'bleu': 1.0, 'precisions': [1.0, 1.0, 1.0, 1.0], 'brevity_penalty': 1.0, 'length_ratio': 1.1666666666666667, 'translation_length': 7, 'reference_length': 6}
 ```
 
 ### Inputs
-- **predictions** (`list` of `list`s): Translations to score. Each translation should be tokenized into a list of tokens.
-- **references** (`list` of `list`s): references for each translation. Each reference should be tokenized into a list of tokens.
+- **predictions** (`list` of `str`s): Translations to score.
+- **references** (`list` of `list`s of `str`s): references for each translation.
+- ** tokenizer** : approach used for standardizing `predictions` and `references`.
+    The default tokenizer is `tokenizer_13a`, a relatively minimal tokenization approach that is however equivalent to `mteval-v13a`, used by WMT.
+    This can be replaced by another tokenizer from a source such as [SacreBLEU](https://github.com/mjpost/sacrebleu/tree/master/sacrebleu/tokenizers).
+
+The default tokenizer is based on whitespace (using the `string.split()` function). It can be replaced by any function that takes a string as input and returns a list of tokens as output. E.g. `word_tokenize()` from [NLTK](https://www.nltk.org/api/nltk.tokenize.html) or pretrained tokenizers from the [Tokenizers library](https://huggingface.co/docs/tokenizers/index).
```

Comment:
I would rephrase it this way, otherwise people may interpret this as if the tokenizer was only doing `.split()`
```suggestion
The default tokenizer is based on whitespace and regexes. It can be replaced by any function that takes a string as input and returns a list of tokens as output. E.g. `word_tokenize()` from [NLTK](https://www.nltk.org/api/nltk.tokenize.html) or pretrained tokenizers from the [Tokenizers library](https://huggingface.co/docs/tokenizers/index).
```